Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -324,7 +324,8 @@ public boolean getBluetoothPresent() {
         try {
             BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter != null) {
-                return bluetoothAdapter.isEnabled();
+				//noinspection ResourceType
```

Comment:
Ah, never mind, it isn't required. This method is actually checking for it. Sorry, not super familiar with the Android SDK.